x-pack/test/security_solution_api_integration/test_suites/detections_response/rules_management/rule_creation/basic_license_essentials_tier/create_rules_bulk.ts

```diff
@@ -38,7 +38,7 @@ export default ({ getService }: FtrProviderContext): void => {
   const auditbeatPath = dataPathBuilder.getPath('auditbeat/hosts');
   const utils = getService('securitySolutionUtils');
 
-  describe('@ess @serverless create_rules_bulk', () => {
+  describe('@ess @serverless @serverlessQA create_rules_bulk', () => {
```

Comment:
My understanding is that this endpoint will be deprecated in 9.0.

Do you think that could have any impact on our decision to add it to the QA test gate?

Reply:
You're right. This test shouldn't be enabled in MKI.